src/openforms/logging/logevent.py

```diff
@@ -348,3 +353,27 @@ def appointment_cancel_failure(appointment: "AppointmentInfo", plugin, error):
         plugin=plugin,
         error=error,
     )
+
+
+def view_submission_details_admin(submission: "Submission"):
```

Comment:
I would re-order the naming scheme so we get a bit of a namespace/grouping (from large to small terms), so if you put all of the events in a list related things group together

`view_submission_details_admin` -> `submission_details_view_admin`

`export_submissions` -> `submission_export_list`